io-engine/src/bdev/nexus/nexus_module.rs

```diff
@@ -21,13 +21,21 @@ pub(crate) struct NexusModule {}
 
 impl NexusModule {
     /// Returns Nexus Bdev module instance.
+    /// # Warning
     /// Panics if the Nexus module was not registered.
     pub fn current() -> BdevModule {
         match BdevModule::find_by_name(NEXUS_MODULE_NAME) {
             Ok(m) => m,
             Err(err) => panic!("{}", err),
         }
     }
+    /// Returns Nexus Bdev module instance, if registered.
+    pub fn current_opt() -> Option<BdevModule> {
+        match BdevModule::find_by_name(NEXUS_MODULE_NAME) {
```

Comment:
IMHO, you can use just '.ok()' method to get the resulting Option:
```
BdevModule::find_by_name(NEXUS_MODULE_NAME).ok()
```

Reply:
yes, indeed